TypeScript/webpack.main.config.ts

```diff
@@ -12,6 +13,16 @@ export const mainConfig: Configuration = {
   module: {
     rules,
   },
+  //can cache groups of components later?
```

Comment:
What does this mean?

Reply:
so it is a comment for me for webpacks ability to cache groups of components based on basic rule sets. So for example all ux-components or pages etc can be stored together. Which can lead to improvements in performance.